cmake/Config.cmake.in

```diff
@@ -0,0 +1,6 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
+find_dependency(OpenSSL)
```

Comment:
I'm not sure if it will result in error but logically it should be before loading targets.